src/lib/ifaces/mac_v.rs

```diff
@@ -8,24 +8,24 @@ use netlink_packet_route::rtnl::nlas::NlasIterator;
 use serde_derive::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-const ETH_ALEN: usize = 6;
+pub const ETH_ALEN: usize = 6;
 
-const MACVLAN_MODE_PRIVATE: u32 = 1;
-const MACVLAN_MODE_VEPA: u32 = 2;
-const MACVLAN_MODE_BRIDGE: u32 = 4;
-const MACVLAN_MODE_PASSTHRU: u32 = 8;
-const MACVLAN_MODE_SOURCE: u32 = 16;
+pub const MACV_MODE_PRIVATE: u32 = 1;
+pub const MACV_MODE_VEPA: u32 = 2;
+pub const MACV_MODE_BRIDGE: u32 = 4;
+pub const MACV_MODE_PASSTHRU: u32 = 8;
+pub const MACV_MODE_SOURCE: u32 = 16;
 
-const IFLA_MACVLAN_MODE: u16 = 1;
-const IFLA_MACVLAN_FLAGS: u16 = 2;
+pub const IFLA_MACV_MODE: u16 = 1;
+pub const IFLA_MACV_FLAGS: u16 = 2;
 // const IFLA_MACVLAN_MACADDR_MODE: u16 = 3;    // not used in query
-const IFLA_MACVLAN_MACADDR: u16 = 4;
-const IFLA_MACVLAN_MACADDR_DATA: u16 = 5;
-const IFLA_MACVLAN_MACADDR_COUNT: u16 = 6;
+pub const IFLA_MACV_MACADDR: u16 = 4;
+pub const IFLA_MACV_MACADDR_DATA: u16 = 5;
+pub const IFLA_MACV_MACADDR_COUNT: u16 = 6;
```

Comment:
There is no need for use to expose these constants. Please remove the `pub`.

Reply:
Done, thanks!